meta-resin-common/recipes-support/resin-mounts/resin-mounts/mnt-boot.mount

```diff
@@ -7,7 +7,7 @@ Conflicts=umount.target
 [Mount]
 What=/dev/disk/by-label/resin-boot
 Where=/mnt/boot
-Type=vfat
+Type=ubifs
```

Comment:
@dlx, we have addressed this issue with this PR: https://github.com/resin-os/meta-resin/pull/744
Could you please resend your PR but leave the *.mount files unmodified?